src/seedu/addressbook/data/person/Name.java

```diff
@@ -39,8 +39,8 @@ public static boolean isValidName(String test) {
     /**
      * Retrieves a listing of every word in the name, in order.
      */
-    public List<String> getWordsInName() {
-        return Arrays.asList(fullName.split("\\s+"));
+    public List<String> getWordsInLowerCase() {
+        return Arrays.asList(fullName.toLowerCase().split("\\s+"));
```

Comment:
I don't think it's a good idea to put the `toLowerCase` here. Since you want to make the find case insensitive, then it might be better just to change FindCommand class.